assets/src/scripts/monitoring-location/components/hydrograph/data-table.test.js

```diff
@@ -3,150 +3,12 @@ import {select} from 'd3-selection';
 import {configureStore} from 'ml/store';
 
 import {drawDataTables} from './data-table';
+import {TEST_PRIMARY_IV_DATA, TEST_GW_LEVELS} from './mock-hydrograph-state';
```

Comment:
I put test data that could be reusued for javascript modules within components/hydrograph in mock-hydrograph-state. I think this should make it simpler to update tests as stored data changes.

Reply:
Good idea. That will come in handy.